### ISSUE TITLE
As a consumer of the operator, I would like to have less steps to manually deploy the operator outside of OLM

### ISSUE THREAD

**Issue by whowutwut** [IBM/ibm-spectrum-scale-csi-operator#74]
**Is your feature request related to a problem? Please describe.**

Create an easier way for the end user to deploy the operator manually when outside OLM (Operator Lifecycle Management).   The proposed solution should address the following: 

1. DO NOT require the user to `git clone` this REPO
2. If the user wants to get the latest released container image from us, there should be no changed required to deploy

**Describe the solution you'd like**

Currently the process is: 

## Apply
```
kubectl apply -f deploy/namespace.yaml
kubectl apply -f deploy/service_account.yaml
kubectl apply -f deploy/role.yaml
kubectl apply -f deploy/role_binding.yaml
kubectl apply -f deploy/crds/ibm_v1alpha1_csiscaleoperator_crd.yaml
kubectl apply -f deploy/operator.yaml
```
##  Delete
```
kubectl delete -f deploy/spectrum_scale.yaml
kubectl delete -f deploy/operator.yaml
kubectl delete -f deploy/role.yaml
kubectl delete -f deploy/role_binding.yaml
kubectl delete -f deploy/service_account.yaml
kubectl delete -f deploy/crds/ibm_v1alpha1_csiscaleoperator_crd.yaml
kubectl delete -f deploy/namespace.yaml
```

Let's take out namespace, since that needs  to be created first and removed last. 

So the proposed flow is: 

### Get and apply the namespace 
1. `kubectl/oc apply  -f http://raw.github..../namespace.yaml` (or save first it, then apply)

### Get and apply the operator 
2 `kubectl/oc apply -f https://raw.github..../installer/generated/ibm-spectrum-scale-csi-operator.yaml` (or save it first, then apply) 

### Get and apply the driver 
3. `curl https://raw.github.../example/spectrum_scale.yaml`
4. Make changes to above file to customize to scale environment 
5. kubectl/oc apply -f spectrum_scale.yaml 

## Delete 

We would want to save it first in the above to more easily delete the driver/operator.  

The opposite of the apply.. 
```
kubectl/oc delete -f spectrum_scale.yaml  # remove driver 
kubectl/oc delete -f ibm-spectrum-scale-csi-operator.yaml  # remove operator
kubectl/oc delete -f namespace.yaml # remove namespace
```

**Describe alternatives you've considered**

**Additional context**

**Comment by gandhisanjayv** [comment]
we should provide steps  install CSI-driver  tools and examples.
tools/generate_pv_yaml.sh
tools/spectrum-scale-driver-snap.sh
[root@viserion-inf ibm-spectrum-scale-csi-driver]# find examples/
examples/
examples/dynamic
examples/dynamic/fileset
examples/dynamic/fileset/podfileset.yaml
examples/dynamic/fileset/pvcfileset.yaml
examples/dynamic/fileset/storageclassfileset.yaml
examples/dynamic/fileset/storageclassfileset_dependent.yaml
examples/dynamic/lightweight
examples/dynamic/lightweight/podlw.yaml
examples/dynamic/lightweight/pvclw.yaml
examples/dynamic/lightweight/storageclasslw.yaml
examples/static
examples/static/static_pod.yaml
examples/static/static_pv.yaml
examples/static/static_pvc.yaml

**Comment by mew2057** [comment]
Closing as this was implemented in 1.0.0
